pkg/network/namescheme/networknamescheme.go

```diff
@@ -20,19 +20,26 @@
 package namescheme
 
 import (
+	"crypto/sha256"
 	"fmt"
+	"io"
 
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 )
 
-const PrimaryPodInterfaceName = "eth0"
+const (
+	// maxIfaceNameLen equals max kernel interface name len (15) - length("-nic")
+	// which is the suffix used for the bridge binding interface with IPAM.
+	// (the interface created to hold the pod's IP address - and thus appease CNI).
+	maxIfaceNameLen         = 11
+	PrimaryPodInterfaceName = "eth0"
+)
 
 // CreateNetworkNameScheme iterates over the VMI's Networks, and creates for each a pod interface name.
 // The returned map associates between the network name and the generated pod interface name.
-// Primary network will use "eth0" and the secondary ones will use "net<id>" format, where id is an enumeration
-// from 1 to n.
+// Primary network will use "eth0" and the secondary ones will be named with the hashed the network name (sha256).
```

Comment:
how is sha256 redundant ? Where is that stated other than here ?

Reply:
I do not see the need to embed in the description which specific algorithm the hash uses.
How is that helpful?

Reply:
I was taking "offense" at the word redundant. It is not redundant, but, it might be useless. 

You claim it isn't useful. I really don't have an opinion.